Quadriga/src/main/webapp/WEB-INF/views/transformation/transformation.jsp

```diff
@@ -0,0 +1,223 @@
+        		<label>
+      				<input type="checkbox" value="${project.projectName}"  name="project" class="projectList"> 
+        		</label>
+        		<a role="button" data-toggle="collapse" data-parent="#accordion" href="#${project}" aria-expanded="true" aria-controls="${project}">
+    			${project.projectName}
+    			</a> 				
+ 				</div>       		
+      		</h4>
+    	</div>
+		<div id="${project.projectName}" class="panel-collapse collapse in" role="tabpanel" aria-labelledby="headingOne">
+      		<div class="panel-body">   				
+   				<ul class="networkToggleList">
+					<li>
+						<details>
+							<ul>							
+								<div class="container-fluid">
+								<div class="row">
+								<c:if test="${not empty networkMap[project.projectName]}">
+								<c:forEach var="network" items="${networkMap[project.projectName]}">	
+									 <div class="col-md-4">
+									 <summary>
+										<input type="checkbox" >
+										<a href="${pageContext.servletContext.contextPath}/auth/editing/visualize/${network.networkId}">
+										<c:out value="${network.networkName}"></c:out>
+										</a>
+									</summary>
+									<li>Workspace: <c:out	value="${network.networkWorkspace.workspace.workspaceName}"></c:out></li>
+									<li>Submitted by: <c:out value="${network.creator.userName}"></c:out>
+									</li>
+									<br>
+									</div>
+								</c:forEach>
+								</c:if>
+								</div>
+								</div>			
+							</ul>
+						</details>
+					</li>
+				</ul>
+       		</div>
+    	</div>
+  </div>
+
+</c:forEach>
+</ul>
+</c:when>
+<c:otherwise>
+		<spring:message code="empty.networks" />
+	</c:otherwise>
+</c:choose>
+</div>
+
+<div class="row">
+<div class="col-md-8"><span class="byline"><input type="checkbox" id="selectAllTransformations">  List of available Transformations.</span></div>
+</div>
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+<c:choose>
+	<c:when test="${not empty dummyTransformations}">
+		<ul class="pagination1">
+			<c:forEach var="transformations" items="${dummyTransformations}">
+				<div class="panel panel-default">
+    				<div class="panel-heading" role="tab" id="headingTwo">
+      				<h4 class="panel-title">   
+						<li>
+							<input type="checkbox" value="${transformations}" name="transformation" class="transformationList">
+							&nbsp;&nbsp;&nbsp;&nbsp;<c:out value="${transformations}"></c:out>
+						</li>
+		    		</h4>
+    				</div>
+				</div>				
+			</c:forEach>
+		</ul>
+	</c:when>	
+</c:choose>
+</div>
+
+<input type="button" value='Submit Project and Transformations' id="confirmationTransformation"/>
+<div id="confirm" style="display:none;">
+	<center><h4>Are you sure you want to transform the following <br>networks with below transformation files?</h4> </center>   
+	<p id="contentholder">
+ 	</p>
+</div>
+
+<div id="alert" style="display:none;">
+	<center><h4>No Transformations or Projects selected</h4> </center>   
```

Comment:
should be lowercase for "transformations" and "projects". also missing a period at the end.